### ISSUE TITLE
Poor accuracy with pretrained_word_embeddings example

### ISSUE THREAD

**Issue by shammimandya** [fchollet/keras#3664]
When I run the tutorial to use pre-trained word embeddings (available in pretrained_word_embeddings.py), I get the following results, finally resulting in a poor accuracy. I am not sure how to fix this problem. Can anyone please help me to fix this problem. Also I am running keras with Theano backend on CPU. Is this something to do with running keras on CPU as against GPU.

Epoch 1/2

128/15998 [..............................] - ETA: 165s - loss: 2.9967 - acc: 0.0625
256/15998 [..............................] - ETA: 150s - loss: nan - acc: 0.0508
384/15998 [..............................] - ETA: 145s - loss: nan - acc: 0.0573
512/15998 [..............................] - ETA: 141s - loss: nan - acc: 0.0566
...

**Comment by esube** [comment]
Your loss is 'nan'. What that means is a lot of things. It could be because of a lot of things and one of them is very high initial learning rate. Post your code or part of your code if you need further help.

**Comment by peterwilli** [comment]
@esube Hi there, I also have loss NaN. I dont know why, but I used the original example without any modifications (except path to files for training)

Here is my output:
```
Using Theano backend.
Indexing word vectors.
Found 400001 word vectors.
Processing text dataset
Found 19997 texts.
Found 214909 unique tokens.
Shape of data tensor: (19997, 1000)
Shape of label tensor: (19997, 20)
Preparing embedding matrix.
Training model.
Train on 15998 samples, validate on 3999 samples
Epoch 1/2
  256/15998 [..............................] - ETA: 573s - loss: nan - acc: 0.0508   
```

**Comment by peterwilli** [comment]
@shammimandya @esube New update: I had sucess (no NaN loss) when I switched from theano to tensorflow backend! I dont know why, but maybe someone can explain :)

**Comment by shammimandya** [comment]
True. I too had success (results as expected) by switching to tensorflow. Thanks @peterwilli for pointing me to this.
